app/models/user.rb

```diff
@@ -1,5 +1,27 @@
 class User < ApplicationRecord
+
+  has_many :viewing_party_users
+  has_many :viewing_parties
+
+  
   validates_presence_of :name
   validates :email, presence: true, uniqueness: true
 
+  def self.all_except(user)
+    where.not(id: user)
+  end
+
+  def name_and_email
+    "#{name} (#{email})"
+  end
+
+  def invited
+    self.viewing_party_users.map do |guest|
+      guest.viewing_party
+    end
```

Comment:
I think utilizing the .through association for one and the chained association for the other method here is something I'm very proud of